build_web_compilers/lib/src/dev_compiler_bootstrap.dart

```diff
@@ -95,6 +95,7 @@ Future<List<Module>> _ensureTransitiveModules(
   // Check that each module is readable, and warn otherwise.
   await Future.wait(jsModules.map((jsId) async {
     if (await reader.canRead(jsId)) return;
+    await reader.canRead(jsId.addExtension('.errors'));
     log.warning(
         'Unable to read $jsId, check your console for compilation errors.');
```

Comment:
Should we mention where to find the `.errors` files in case this was run without `--rerun-failing-actions` and so the message might not be on the console?

Reply:
Sure ya, I like that idea